kahuna/public/js/components/gr-tooltip/gr-tooltip.js

```diff
@@ -3,23 +3,26 @@ import 'titip';
 
 export const tooltip = angular.module('grTooltip', []);
 
-tooltip.directive('grTooltip', ['onValChange', function (onValChange) {
-    return {
-        restrict: 'A',
-        link: function ($scope, element, attrs) {
-            const position = attrs.grTooltipPosition || 'bottom';
+tooltip.directive('grTooltip', [
+    'onValChange',
+    '$interpolate',
+    function (onValChange, $interpolate) {
+        return {
+            restrict: 'A',
+            link: function ($scope, element, attrs) {
+                const position = attrs.grTooltipPosition || 'bottom';
+                const toolTipText = $interpolate(attrs.grTooltip);
```

Comment:
It tells angular to parse the tooltip. This was so I could put scope variables in the tooltip, like I wanted to put the lease creator or notes etc. and it was printing "{{lease.notes}}" or w.e.